src/api/app/controllers/application_controller.rb

```diff
@@ -476,4 +478,8 @@ def forward_from_backend(path)
   def shutup_rails
     Rails.cache.silence! unless Rails.env.development?
   end
+
+  def load_config
+    @configuration = ::Configuration.first
```

Comment:
I know that we do this for every WebUI call... but is this needed for every API call?